### ISSUE TITLE
Courier AI has run into an issue

### ISSUE THREAD

**Issue by setdebarr** [ldtteam/minecolonies#6400]
### Prerequisites 

- [ ] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [ ] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version: 1.16.4
- MineColonies Version: 0.13.557 Alpha
- Structurize Version: 0.13.104 Alpha

## Problem description

I'm running a Direwolf20 1.16 v1.3.1 server and a player sent me a screenshot of their courier saying there was a problem with the Worker's AI and requesting that I send the developer team my latest.log file, which I've attached.
[latest.log](https://github.com/ldtteam/minecolonies/files/5817212/latest.log)

### Notes or related things

---
## Viewers

* Add a thumbs-up to the issue to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by ravenbuilder934** [comment]
log in a gist: https://gist.github.com/ravenbuilder934/1f046139c19146fdfe8e7b410be2defa
(please put it in a gist next time)

**Comment by Qrsmith** [comment]
I ran into the same situation on a 1.15.2 server that I run.
![image](https://user-images.githubusercontent.com/49616009/105069443-f38dba00-5a4f-11eb-9cd7-9f29f9b61237.png)
[latest.log](https://github.com/ldtteam/minecolonies/files/5837456/latest.log)

**Comment by Leginiel** [comment]
I got this message on my server, too (1.16.4 / minecolonies 0.13.535 ALPHA / sturctur0.13.104 Alpha). 
Additional I was able to trace back the reason, why this issue might happen:

Reproduction steps, as far as I could trace them down
- I created the build request to deconstruct my blacksmith (medievalspruce, lvl 1)
- The builder finished deconstructing the blacksmith 
- I'll assume that there were some open request for the blacksmith still preset in time the building was deconstructed

I could see in the latest.log shortly after the exception, that both courier failed near the location where the old controller block of the blacksmith was located. 

I will try to verify it tomorrow with the newest version of minecolonies and structurize.

You'll find the latest.log here (taken like 10 Minutes after the exception) 
https://gist.github.com/Leginiel/983752f98db382263ea564beee85f377
